### ISSUE TITLE
Is there a support forum for this?

### ISSUE THREAD

**Issue by kahanu** [MSOpenTech/AngularCloudDataConnector#5]
I don't believe I'm this much of a moron, but I tried to recreate your functionality in a new app just to see if I could get it to work, and I can't.

I just created a new blank website in Visual Studio 2013 with one html file and one app.js file.  I literally copied and pasted your demoApp module into my app.js file, and created a barebones HomeController that just had the $scope.initialize() method.  The html is just supposed to display the firstname. I even kept your url and key, and I get the error:

"The requested database object could not be found."

I do have all the angular script references and the CDC references, but I'm obviously missing something.

I'm trying to start a new mobile app using angular and I want to use Azure Mobile Services.  Any ideas why I can't even connect to your mobile service in my app using this CDC helper?

**Comment by deltakosh** [comment]
Hey and thank you for using angularCDC:)

Can you please share your code ? It would be easier to find what's missing.

**Comment by kahanu** [comment]
Thank you for assisting me on this.  I really want to use your plugin in my project.

I zipped up my Visual Studio 2013 website to my web site to download.  http://www.kingwilder.com/downloads/angularcdctest.zip - 67KB

I know I'm probably missing something blatantly obvious but I can't find what it is.  Again thank you.

**Comment by deltakosh** [comment]
Are you using Win10 preview? If so there is an issue with IDB on IE. This code works for me on Chrome or on IE on win8.1

**Comment by kahanu** [comment]
Well that's frustrating. I just tried it again and it gives me the same error message that the database object could not be found.  I'm using the Visual Studio web server.  And no I'm on Windows 7 Ultimate.  I'm at a loss.

**Comment by deltakosh** [comment]
Are you using ie11? Could you try to remove idb database in the settings pane?

**Comment by kahanu** [comment]
No, I was using Firefox v35.  I just tried it in IE 11 and Chrome and it worked in both browsers!!!  So what's up with Firefox?

**Comment by kahanu** [comment]
My intent is to use this in a mobile hybrid app (Cordova), that would that make a difference then?

**Comment by deltakosh** [comment]
Yes definitely! This should work directly. I'm assuming Firefox may have issue with IDB
